### DISCUSSION TITLE
Lencense Migration

### DISCUSSION THREAD

**Discussion by Victor-Hugo-Aquino** [aclap-dev/video-downloadhelper#1037] [License & Purchase requests]
### What's your issue?

Problem description:

Hello, support team! I'm facing a rather complicated situation with my recent purchase. I purchased a license for the extension for Google Chrome, but realized too late that I had made a mistake. In reality, I use Firefox as my main browser and that's where I need the extension.
Request:

I would like to know if there is a possibility of transferring my license key. Would it be feasible to convert the license I purchased for Chrome into a license compatible with Firefox? I understand that they are different platforms, but I hope there is a solution that doesn't involve canceling the current purchase and making a new one.
If it's relevant to the process, I can provide the first digits of my license key:

### The 10 first digits (only 10) of your license number or email address

ffaee912cb

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Sorry we can't swap licenses. Chrome and Firefox premium are separate products with different prices and features.
What I can do is refund your Chrome premium so that you can purchase Firefox premium if you want to.
Do you want me to do this ?

**Reply by Victor-Hugo-Aquino** [reply]
Yes

**Reply by jcfrog** [reply]
You have been refunded. Should be available in a few business days.
